guides/blending/client-side-blending/README.md

```diff
@@ -0,0 +1,52 @@
+# Client-Side Blending
+
+When merging responses from different apps with [server-side blending](server-side-blending), the views are stacked on top of each other. In most cases, that's not what we want. Instead, we would like to compose elements in the views to make the result look like one page. That's what client-side blending does:
+
+![Blending comparison](/assets/BlendingComparison2.png)
+
+## Composing Views
+ 
+To solve this, we could edit the source code of the apps involved. This is impractical, and in some cases even impossible. The main reason is that the apps in a system, or application suite, are often created by different developers and it's not always possible to get access to their source code. A secondary reason is versatility. If an app is modified to work well with your application suite, it might not work well with other application suites.
+
+Client-side blending lets you change the structure of blended apps without touching the source code. It does this by swapping the default composition of HTML that's sent to the client with an alternative composition. This requires a couple of things:
```

Comment:
"change the structure of .. apps" we are not changing the structure of apps, we are changing their look/feel/layout